src/plugins/data-card/use-sortable-card-style.ts

```diff
@@ -0,0 +1,52 @@
+import classNames from "classnames";
+interface IProps {
+  transform?: { x: number; y: number } | null;
+  indexInStack: number;
+  stackIsExpanded: boolean;
+}
+
+function getTiltAngle(index: number, stackIsExpanded: boolean) {
+  if (stackIsExpanded) return 0;
+  const angles = [0, -4, 3, -3, 3];
+  return angles[index % angles.length];
+}
+
+function getOpacity(index: number) {
+  const gradualFade = 1 - (index - 2) * 0.25;
+  return index > 2 ? gradualFade : 1;
+}
+
+export const useSortableCardStyles = (props: IProps) => {
+  const { transform, indexInStack, stackIsExpanded } = props;
+
+  let dynamicStyles;
+
+  const dynamicClasses = classNames(
+    "drag-handle", "sortable", "card",
+    {
+      "in-expanded-stack": stackIsExpanded,
+      "in-collapsed-stack": !stackIsExpanded
+    },
+  );
+
+  if (transform) {
+    dynamicStyles = {
+      transform: `translate3d(${transform.x}px, ${transform.y - 25}px, 0)`,
```

Comment:
Why do we need 3d transforms here?  I'm unsure what this is doing, maybe a comment would help.